api-tester/src/main/java/com/revenuecat/apitester/java/PeriodAPI.java

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.apitester.java;
+
+import com.revenuecat.purchases.models.Period;
+
+@SuppressWarnings({"unused", "SpellCheckingInspection"})
+final class PeriodAPI {
+    static void check(final Period period) {
+        int val = period.getValue();
+        Period.Unit unit = period.getUnit();
+        String iso8601 = period.getIso8601();
+
+        switch (unit) {
+            case DAY:
+            case WEEK:
+            case MONTH:
+            case YEAR:
+            case UNKNOWN:
+        }
```

Comment:
It allows us to "safely" to return an nullable `Period` when creating one from a ISO 8601 string 😬 

I didn't love that we went from a `period: String` to `period: Period?` so having an `UNKNOWN` type since we are parsing it seemed like the least worst option 🤷‍♂️

Reply:
Sorry, I think my comment wasn't clear... I think the assumption is that defining new enum values will somehow create a compiletime error because the switch isn't complete anymore? I don't think the java compiler does that by default (am I missing something?). So in general I was asking what the purpose of this empty swith/case would be.